### DISCUSSION TITLE
Update from 0.9.4 to 1.0.1

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#3633] [Q&A] [answered]
Hello,

I'm currently updating to 1.0.1 coming from 0.9.4. What happened to TextEdit "ChangeTextOnKeyPress" Property? My current default options are:
```

builder.Services.AddBlazorise(options =>
{
    options.Debounce = true;
    options.DebounceInterval = 300;
})
.AddBootstrap5Providers();
```

But for some TextEdit's I need to set `ChangeTextOnKeyPress="false"`. How is that possible?

Thanks for help :).

**Comment by David-Moreira** [comment, marked as answer]
Hello,
We always have migration notes in our major releases.

Please refer to : https://blazorise.com/news/release-notes/100

You are probably looking for this:
![image](https://user-images.githubusercontent.com/22283161/160188204-5a72d18d-228f-42c7-b574-43c64ac61b7b.png)

**Reply by stsrki** [reply]
Also, look for the 0.9.5 migration part in https://blazorise.com/news/release-notes/095

PS. please accept the answer if it helps you.

**Reply by NeoCoderMatrix86** [reply]
Thanks, perfect help :)
